### PR TITLE
Remove unused usage of ftm

### PR DESCRIPTION
**Pull request by soniasingla**
Signed-off-by: Sonia Singla <soniasingla.1812@gmail.com>

### CODE REVIEW

**Review by sonasingh46 — APPROVED** [review]

File: pkg/pool/operations/create.go [hunk 1]
```diff
@@ -59,7 +59,6 @@ func (oc *OperationsConfig) Create(cspi *cstor.CStorPoolInstance) error {
 		}
 		// Remove this raidGroup
 		cspiCopy.Spec.DataRaidGroups = append(cspiCopy.Spec.DataRaidGroups[:i], cspiCopy.Spec.DataRaidGroups[i+1:]...)
-		break
```
Comment (mittachaitu) [inline pkg/pool/operations/create.go:62]: This `break` is required
 
**Context**: When the user specifies multiple `DataRaidGroups` first we will create a pool(at line 55 using 0th index raid group) and we will remove used RaidGroup from the total group list and then expand the pool.

**Review by mittachaitu — CHANGES_REQUESTED** [review]
Provided comments PTAL

**Comment by mittachaitu** [comment]
@soniasingla Did you get a chance to incorporate review comments?

**Comment by soniasingla** [comment]
@mittachaitu done sorry for delay

**Pull request merged**